projects/fb-openlayers/src/lib/resources/layer-charts.component.ts

```diff
@@ -252,8 +254,11 @@ export class FreeboardChartLayerComponent
                 );
               } else {
                 // mbtiles source
+                let isS57 = charts[i][1].type == "S-57"
                 const source = new VectorTileSource({
                   url: charts[i][1].url,
+                  minZoom: isS57 ? charts[i][1].minZoom : null,
```

Comment:
use minZ instead of null

Reply:
I think minzoom and maxzoom of the vector source should be as generated  ( configured)  they have another purpose then minzoom and maxzoom of the vector layer